src/Mutation/FileMutationGenerator.php

```diff
@@ -85,21 +85,18 @@ public function generate(
         Assert::allIsInstanceOf($mutators, Mutator::class);
         Assert::allIsInstanceOf($nodeIgnorers, NodeIgnorer::class);
 
-        $filePath = $fileInfo->getRealPath() === false
-            ? $fileInfo->getPathname()
-            : $fileInfo->getRealPath()
-        ;
-
         if ($onlyCovered && !$codeCoverage->hasTests()) {
             return;
         }
 
-        $initialStatements = $this->parser->parse($fileInfo);
+        $sourceFilePath = $fileData->getSplFileInfo()->getPathname();
+
+        $initialStatements = $this->parser->parse($fileData->getSplFileInfo());
 
         $mutationsCollectorVisitor = new MutationsCollectorVisitor(
             new NodeMutationGenerator(
                 $mutators,
-                $filePath,
+                $sourceFilePath,
```

Comment:
Minor change to avoid using `getSplFileInfo()` two times

```suggestion
        $fileInfo = $fileData->getSplFileInfo();

        $initialStatements = $this->parser->parse($fileInfo);

        $mutationsCollectorVisitor = new MutationsCollectorVisitor(
            new NodeMutationGenerator(
                $mutators,
                $fileInfo->getPathname(),
                $initialStatements,
                $codeCoverage,
                $onlyCovered,
                $this->lineRangeCalculator
            )
        );
```

Reply:
Fixed